### PR TITLE
Add custom class names support

### PR DESCRIPTION
**Pull request by bogdan-stefan-hs**
These changes add support for passing the resizer classes via props.

### CODE REVIEW

**Comment by bogdan-stefan-hs** [comment]
@catalin-miron-hs These changes allow custom classes, however the tests fail now, as they search the DOM for the exact 'Resizer', 'Pane' and 'SplitPane' classes.

**Comment by catalin-miron-hs** [comment]
@bogdan-stefan-hs yes, the tests are failing because you changed the functionality. Please update the test that search for `classNames` in DOM.

**Comment by bogdan-stefan-hs** [comment]
@catalin-miron-hs I fixed the tests by allowing a custom classname for Pane and Resizer too. Is this ok? Also, can I get a code review from you?

File: package.json [hunk 1]
```diff
@@ -27,6 +27,7 @@
   "devDependencies": {
     "babel": "5.8.23",
     "babel-eslint": "4.1.3",
+    "babel-loader": "^6.2.4",
```
Comment (andrei-picus-hs) [inline package.json]: Is this really needed?

Comment (bogdan-stefan-hs) [inline package.json]: Not really, it was a npm link dev issue. Removing it.

**Comment by bogdan-stefan-hs** [comment]
@valentin-radulescu-hs @flavius-tirnacop-hs please review

**Comment by flavius-tirnacop-hs** [comment]
LGTM

File: package.json [hunk 2]
```diff
@@ -1,18 +1,18 @@
...
   },
   "license": "MIT",
   "bugs": {
-    "url": "https://github.com/tomkp/react-split-pane"
+    "url": "https://github.com/ubervu/react-split-pane"
   },
-  "homepage": "https://github.com/tomkp/react-split-pane",
-  "author": "tomkp <tom@tomkp.com>",
+  "homepage": "https://github.com/ubervu/react-split-pane",
+  "author": "UberVU",
```
Comment (valentin-radulescu-hs) [inline package.json:15]: Can we keep the original author name?

Comment (flavius-tirnacop-hs) [inline package.json:15]: Why lol?

Comment (bogdan-stefan-hs) [inline package.json:15]: I found 2 forks of this, [this one](https://github.com/rohn/react-split-pane/blob/master/package.json) keeps it as it is, [this one](https://github.com/kadirahq/react-split-pane/blob/master/package.json) modifies it.
I guess nobody cares about it.

Comment (valentin-radulescu-hs) [inline package.json:15]: Fair enough, we can leave it as is

**Comment by valentin-radulescu-hs** [comment]
LGTM

**Pull request merged**